matcher/matcher/tasks.py

```diff
@@ -35,14 +35,18 @@ def do_match(
     with open(config_path) as f:
         config = yaml.load(f)
 
-    # Initializing: let's get started by collecting some job metadata
+    # Initializing: let's get started by collecting and logging some job metadata
     metadata = {
         'match_job_start_time': datetime.datetime.now(),
         'match_job_id': utils.unique_match_job_id(),
         'base_data_directory': base_data_directory,
         'config': config
     }
-    logger.info("Matching process started!")
+    logger.info("Matching process started with the following configuration:")
```

Comment:
Less important: Just loop over the key/value pairs and log each one to make it future-proof (either new or renamed keys)

More important: Include a common prefix (e.g. 'matching config') to each line so the matcher config is easily greppable